index.html

```diff
@@ -56,6 +56,9 @@ <h3 class="about__article__title"><i class="about__article__title__icon ion ion-
         <p class="about__article__text">Dobrých lidí s dobrými nápady je vždycky málo, a tak jsme se rozhodli, že by bylo super vás sdružovat.</p>
       </article>
     </section>
+    <section id="calendar">
```

Comment:
`id`cka jsou bad, drz se prosim stejneho pristupu jako je na zbytku stranky

Neni to dokonale, ale ridil bych se necim jako je BEM (https://getbem.com/introduction/ , https://en.bem.info/methodology/css/) at se v tom udrzuje aspon nejaky poradek kdyz to zacne bobtnat.

`<section id="calendar">` => `<section class="calendar">`